code/game/objects/items/toys.dm

```diff
@@ -405,6 +405,8 @@
 /*
  * Crayons
  */
+/mob/living/carbon
+	var/sprayOverlay

```

Comment:
I'd rather this be moved to carbon_defines.dm to be honest (or however that file is called).